src/esp/bindings/SimBindings.cpp

```diff
@@ -68,8 +68,20 @@ void initSimBindings(py::module& m) {
                     &Simulator::setFrustumCullingEnabled,
                     R"(Enable or disable the frustum culling)")
       /* --- Physics functions --- */
+
+      .def("get_template_handle_by_ID", &Simulator::getObjectTemplateHandleByID,
+           "object_id"_a)
+      .def("get_template_handles", &Simulator::getObjectTemplateHandles,
+           "search_str"_a = "")
+      .def("get_file_template_handles",
+           &Simulator::getFileBasedObjectTemplateHandles, "search_str"_a = "")
+      .def("get_prim_template_handles", &Simulator::getPrimitiveTemplateHandles,
+           "search_str"_a = "")
```

Comment:
Since these will be unified soon do we need this distinction in Simulator?

Reply:
Well, as I mentioned earlier, I felt it might have worth getting templates to the non-file-based objects (perhaps instead of calling them "prim_templates" they could be "generated_templates" ?).  Again, not sure if folks would care from a usage perspective, but if so, i wanted to make it available.